frontend/src/main/utils/useBackend.js

```diff
@@ -33,7 +33,7 @@ export function useBackend(queryKey, axiosParameters, initialData) {
             return response.data;
         } catch (e) {
             const errorMessage = `Error communicating with backend via ${axiosParameters.method} on ${axiosParameters.url}`;
-            toast(errorMessage);
+            
```

Comment:
Sorry, that line has to come back.   We do want to make the toasts not appear when the user is not logged in.   But this is not the way to do it.

Instead, a few options:

Option 1:  Disable that toast if and only if you are NOT logged in.
 
Option 2: 

* Add a new home page that only has a login button on it.     Make that be the home page.  The home page does NOT try to get any information from the backend.  (That might mean that it needs a different layout from BasicLayout.)
* After the user is logged in, the home redirects to a "play" page that has the current content of the home page.

Option 3: Make the backend route that gets the list of commons be one that you can get even if you are not logged in.  

Option 4: In App.js if you are not logged in, the `/` route doesn't render the HomePage but instead renders a login page that doesn't try to ask the backend for anything.

There may be other options as well.

But just disabling the toast the tells us when there are backend problems of all kinds is NOT the way to do this.

Reply:
Fixed. I used Option 1. Add login check in useBackend.js to fix this problem

Reply:
I found that the test does not cover line 38, which the error message toast. Get stuck by testing this. Wanna try this later if the points are not enough.